### PR TITLE
=clt #18440 Harden ClusterClientSpec again

### PR DESCRIPTION
**Pull request by patriknw**
The fix in https://github.com/akka/akka/pull/17903 was not using
the correct configuration property. The issue in #17735 was never
fixed by that PR, but now it is.

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #18440

**Comment by patriknw** [comment]
I have also ran this some 250 times during night.

File: akka-cluster-tools/src/multi-jvm/scala/akka/cluster/client/ClusterClientSpec.scala [hunk 1]
```diff
@@ -39,7 +39,7 @@ object ClusterClientSpec extends MultiNodeConfig {
     akka.cluster.client.heartbeat-interval = 1s
     akka.cluster.client.acceptable-heartbeat-pause = 3s
     # number-of-contacts must be >= 4 because we shutdown all but one in the end
-    akka.cluster.client.number-of-contacts = 4
+    akka.cluster.client.receptionist.number-of-contacts = 4
```
Comment (drewhk) [inline akka-cluster-tools/src/multi-jvm/scala/akka/cluster/client/ClusterClientSpec.scala]: :D

**Comment by drewhk** [comment]
LGTM

**Comment by drewhk** [comment]
Great detective work btw.

**Pull request merged**